php/site-utils.php

```diff
@@ -179,12 +207,25 @@ function add_site_redirects( $site_name, $le ) {
 	} else {
 		$site_name_with_www = 'www.' . $site_name;
 		// ee site create example.com --le
-		if ( $le ) {
+		if ( $ssl ) {
 
 			$content = "
 server {
 	listen  80;
+	server_name  $site_name_with_www;
```

Comment:
Can this go in mustache template? It will be easy to read and update config in that.

Reply:
Resolved.